OptimizelySDK/Optimizely.cs

```diff
@@ -361,18 +361,19 @@ public virtual bool IsFeatureEnabled(string featureKey, string userId, UserAttri
                 return false;
             }
 
-            if (decision.Source == FeatureDecision.DECISION_SOURCE_EXPERIMENT)
```

Comment:
why you need to change this whole portion, can you just use 
decision.Source == FeatureDecision.DECISION_SOURCE_EXPERIMENT && decision.Source.Variation.FeatureEnabled
That's it.
@oakbani please verify.
Nothing to change here except above line.

Reply:
It needs futher change because if decision source is experiment and featureEnabled is not true, we also need to return false.